action.yaml

```
@@ -18,15 +18,8 @@ runs:
         GITHUB_TOKEN: ${{ inputs.github-token }}
         RELEASE_BRANCHES: main
         WITH_V: true
-    - name: Get major version
-      id: major-version
-      needs: bump-version
-      shell: bash
-      run: |
-        echo "::set-output name=major-version-tag::$(cut -d . -f 1 <<< ${{steps.bump-version.outputs.new_tag}})"
-    - name: Push major tag
-      needs: major-version
-      uses: anothrNick/github-tag-action@1.36.0
-      env:
-        GITHUB_TOKEN: ${{ inputs.github-token }}
-        CUSTOM_TAG: ${{ steps.major-version.outpus.major-version-tag }}
+    - name: Update major and minor level tags
+      uses: sersoft-gmbh/running-release-tags-action@v2
```

Something Ronny has suggested is to look at how many stars a given action has. This one has very few. I'm not opposed to using it, but I suggest fixing to a specific tag so that future changes upstream don't break our workflow.
```suggestion
      uses: sersoft-gmbh/running-release-tags-action@v2.1.0
```

---

I've committed the change, but I would rather base the decision off of the number of users rather than stars. I've never used the star feature, and I don't think an action would be where people would.